src/goto-cc/Makefile

```diff
@@ -48,7 +48,7 @@ include ../common
 ifeq ($(BUILD_ENV_),MSVC)
 all: goto-cl$(EXEEXT)
 endif
-all: goto-cc$(EXEEXT)
+all: goto-gcc$(EXEEXT)
```

Comment:
Could you please change the lines above so that this is within an `else`/`endif` instead of being done for MSVC as well? Furthermore, please add `goto-gcc$(EXEEXT)` to `CLEANFILES` above.